my-app/src/Components/Account.tsx

```diff
@@ -2,40 +2,25 @@ import React, { useState } from "react";
 import ToolTip from "./ToolTip";
 import AccountHover from "./AccountHover";
 import { useNavigate } from "react-router-dom";
+import { AccountProp } from "../TypesDefines/types";
 
-type Account = {
-  sidepanelOperation: (x: boolean) => void;
-  sidePanel: Boolean;
-};
-
-function Account({ sidepanelOperation, sidePanel }: Account) {
+function Account({ sidepanelMinimize, sidePanel }: AccountProp) {
+  let navigate = useNavigate();
   const [accounthov, setAccounthov] = useState<Boolean>(false);
-  window.addEventListener("click", () => m());
-  function m() {
+  window.addEventListener("click", () => clickHandle());
+  function clickHandle() {
     if (accounthov) {
       setAccounthov(false);
     }
   }
 
-  let navigate = useNavigate();
-
   const accountUser: string = "gokulkr";
```

Comment:
why do we need this variable?

Reply:
there is need an username as prop for the AccountDropDown